fastlane/Pluginfile

```diff
@@ -2,7 +2,9 @@
 #
 # Ensure this file is checked in to source control!
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.18.1'
+gem 'fastlane-plugin-wpmreleasetoolkit',
+  git: 'https://github.com/wordpress-mobile/release-toolkit',
+  branch: 'allow-paths-outside-project-folder'
```

Comment:
I was tired of seeing a whole line diff that I had store horizontally scroll whenever going from a tag to a branch and vice versa... 🤓